software/py/blood_graph_key.py

```diff
@@ -0,0 +1,48 @@
+import sys
+import argparse
+from blood_graph import BloodGraph
+from PIL import Image, ImageDraw, ImageFont
+from itertools import chain
+
+WIDTH  = 256
+HEIGHT = 256
+DEFAULT_MODE = "detailed"
+
+def parse_args():
+    parser = argparse.ArgumentParser(description="Depicts a visual key for bloodgraph")
+    parser.add_argument("--mode", default=DEFAULT_MODE, type=str,
+                        help="Type of bloodgraph - abstract / detailed")
+    args = parser.parse_args()
+
+    if (not (args.mode == "abstract" or args.mode == "detailed")):
+        parser.error("Invalid mode, can be one of abstract / detailed.")
+        sys.exit()
+
+    return args
+
+
+
+if __name__ == "__main__":
+    args = parse_args()
+
+    bg = BloodGraph(0,1,1, args.mode)
+    img = Image.new("RGB", (WIDTH, HEIGHT), "black")
```

Comment:
Why are WIDTH and HEIGHT constants?

Reply:
Changing the font characteristics (i.e. the font height and width) looked like more trouble than it was worth. Given that, I picked a width and height that looked reasonable.